imap_processing/ultra/l1c/ultra_l1c_pset_bins.py

```diff
@@ -112,6 +120,33 @@ def cartesian_to_spherical(
     return np.degrees(az), np.degrees(el), magnitude_v
 
 
+def spherical_to_cartesian(
```

Comment:
I wonder if it would be good to pull this out into a common utility file. Not sure if others will need this though.